extensions/Nuke/Nuke_body.php

```diff
@@ -251,7 +248,7 @@ protected function getNewPages( $username, $limit ) {
 		foreach ( $result as $row ) {
 			$pages[] = array(
 				Title::makeTitle( $row->rc_namespace, $row->rc_title ),
-				$username === '' ? User::getUsername( $row->rc_user, $row->rc_user_text ) : false // SUS-812
+				$username === '' ? User::getUsername( $row->rc_user, inet_ntop( $row->rc_ip_bin ) ) : false // SUS-812
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)

[project-id]: wikia:app